src/platforms/javascript/index.mdx

```diff
@@ -21,121 +21,59 @@ Using a framework? Take a look at our specific guides to get started.
+## Initialize
 
-### Set the Release Version
+`init` Sentry as soon as possible in your app. Once this is done, all unhandled exceptions are automatically captured by Sentry.
 
-When you configure Sentry to include the version of your application, Sentry can tell you about regressions as well as detailed information about the suspect commit. You will also need releases for source maps.
+This snippet includes:
+- the [release](/product/releases/) of your application, using `process.env.npm_package_version`, which both enables [source maps](/platforms/javascript/sourcemaps/) as well as tracking of regressions and suspect commits.
+- our [performance monitoring](/product/performance/) package.
 
-Use the `process.env.npm_package_version`:
+<PlatformContent includePath="initialize-configure" />
 
-```JavaScript
-Sentry.init({
-  dsn: '___PUBLIC_DSN___',
-  release: 'my-project-name@' + process.env.npm_package_version,
-});
-```
+<Alert level="note" title="Note"><markdown>
 
-Learn more about what releases can do in [our documentation for releases](/product/releases/).
+We'll automatically assign you a Data Source Name (DSN), which looks like a standard URL. It's required by Sentry and configures the protocol, public key, server address, and project identifier. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in sentry.io.
 
-### Upload Source Maps
+</markdown></Alert>
 
-We **highly recommend** you incorporate source maps to receive the full benefit of error tracking and monitoring. See our [source maps documentation](/platforms/javascript/sourcemaps/) to learn more.
+## Test
 
-## Monitor Performance
+The snippet below includes an intentional error, so you can test that everything is working as soon as you set it up.
 
-Install performance monitoring using Sentry’s JavaScript SDK using both the `@sentry/browser` and `@sentry/tracing` packages:
+<PlatformContent includePath="test-verify" />
 
-```bash {tabTitle: ESM}
-# Using yarn
-$ yarn add @sentry/browser @sentry/tracing
+To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved. Explore [Performance](/product/performance/) by selecting it from the sidebar, then viewing the homepage.
 
-# Using npm
-$ npm install --save @sentry/browser @sentry/tracing
-```
+## Next Steps
 
-<Alert level="note" title="Note"><markdown>
+- **[Upload Source Maps](/platforms/javascript/sourcemaps/)**
 
-If you are using our CDN, please see our documentation for [installing with CDN](/platforms/javascript/configuration/install-cdn).
+  We **highly recommend** you incorporate source maps to receive the full benefit of error tracking and monitoring.
 
-</markdown></Alert>
+- **[Configure `tracesSampleRate`](/platforms/javascript/configuration/filtering/)**
```

Comment:
This link should probably point to `/platforms/javascript/configuration/filtering/#sampling` That being said, does sampleRate supersede tracesSampleRate?

Reply:
That's a better link, thanks. I don't know the answer to your questions, @HazAT - which sample rate takes precedence?